cluster/src/main/java/io/servicefabric/cluster/ClusterMembershipQualifiers.java

```diff
@@ -1,13 +1,21 @@
 package io.servicefabric.cluster;
 
-import io.servicefabric.cluster.gossip.Gossip;
+import io.servicefabric.transport.TransportHeaders;
 import io.servicefabric.transport.TransportMessage;
+import io.servicefabric.transport.protocol.Message;
 import rx.functions.Func1;
 
-public final class ClusterMembershipQualifiers {
-	public static final String SYNC = "pt.openapi.core.cluster/membership/sync";
-	public static final String SYNC_ACK = "pt.openapi.core.cluster/membership/syncAck";
-	public static final String GOSSIP_MEMBERSHIP = "pt.openapi.core.cluster/membership/gossip";
+final class ClusterMembershipQualifiers {
```

Comment:
Let's have single ```ClusterQualifiers``` or ```ClusterConstants``` class per entire module instead of having several little ones.